cypress/e2e/flow-builder/blocks/moving.cy.ts

```diff
@@ -0,0 +1,74 @@
+import {IBlock} from '@floip/flow-runner'
+import Chainable = Cypress.Chainable
+
+describe('Moving a block', () => {
+  beforeEach(() => {
+    cy.createFlow({
+      label: 'Moving a block',
+    })
+
+    cy.addBlock(['Content', 'Message']).then(blockId => {
+      cy.get(`[data-cy="block--${blockId}"]`)
+        .find('[data-cy^="block-handle--"]')
+        .as('blockArea')
+    })
+
+    // wait for flow validation to succeed and flow validation toolbar to disappear
+    // to avoid interference with block coordinates
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(200)
+  })
+
+  it('moves the block', () => {
+    const deltaX = 200
+    const deltaY = 300
+
+    // eslint-disable-next-line arrow-body-style
+    return getBlockVisualCoordinates().then(({x: oldVisualX, y: oldVisualY}) => {
```

Comment:
I'm wondering if we always need these `return`  on async methods, I didn't use them in mine and it's working https://github.com/FLOIP/flow-builder/pull/494/files#diff-6e06e9c0a89fc89cc254694808341451273525e8839c678f5136bdedd7a4eaacR11

Could you pls try to remove them?

Reply:
Yes, I can try 👍

Reply:
It worked!